setup.cfg

```
@@ -33,6 +33,7 @@ install_requires =
     slackclient==2.5.0  # fixes bug with LinkButtonElement
     pyyaml
     requests
+    sqlalchemy
```

Since you are using PostgreSQL in your connect string, you may want to update this to

```suggestion
    sqlalchemy[postgresql]
```

This will automatically install `psycopg2` when installing `sqlalchemy`.